README.md

```diff
@@ -96,10 +134,19 @@ When you get a URL from ngrok, go ahead and update ViewController.swift with the
 
 For this quick start, the Application transport security settings are set to allow arbitrary HTTP loads for testing your app. For production applications, you'll definitely want to retrieve access tokens over HTTPS/SSL.
 
-## Have fun!
+## More Documentation
+
+You can find more documentation on getting started below:
+
+* [Getting Started](https://www.twilio.com/docs/api/video/getting-started)
+* [Docs](https://media.twiliocdn.com/sdk/ios/video/releases/1.0.0-beta15/docs/index.html)
```

Comment:
The link resolves to the beta14 docs. Don't mind adding it as long it will resolve to the latest GA release.